packages/twenty-front/src/modules/object-record/record-filter/utils/computeViewRecordGqlOperationFilter.ts

```diff
@@ -584,20 +584,38 @@ export const computeFilterRecordGqlOperationFilter = ({
 
       if (options.length === 0) return;
 
+      const noValueOptions = options.filter((opt) => opt === '');
```

Comment:
Can we fix this typescript error ? 

"option" would be better to avoid using abreviation even for anonymous function parameters, as we try to do as much as possible in the codebase.